EventsExpress/ClientApp/src/components/helpers/form-helpers/render-radio-button.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import FormControl from '@material-ui/core/FormControl';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+
+export default ({ input, label, ...rest }) => {
+    return (
+        <FormControl>
+            <RadioGroup {...input} {...rest}>
+                <FormControlLabel value="blocked" control={<Radio />} label="Blocked" />
```

Comment:
Let remake this to receive options from the parent's context, so this one will be more general and reusable